SheepDash/core/src/com/missionbit/game/sprites/Sheep.java

```diff
@@ -1,37 +1,81 @@
 package com.missionbit.game.sprites;
 
+import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
 import com.badlogic.gdx.math.Rectangle;
 import com.badlogic.gdx.math.Vector3;
 
 /**
  * Created by MissionBit on 6/27/17.
  */
 
-public class Sheep {
-    public Vector3 position;
-    private Vector3 velocity;
+public class Sheep extends Animals{
     private Texture sheep;
-    public Rectangle bounds;
+    private Rectangle bounds1;
+    private Animation sheepAnimation;
+    private Texture sheepDead;
+    private Animation sheep2Animation;
 
-    public Sheep(int x, int y){
-        position = new Vector3(x, y, 0);
-        velocity = new Vector3(0,0,0);
-        sheep = new Texture("Sheep(1)");
+    public Sheep(int x, int y) {
+        super(x, y);
+        sheep = new Texture("Sheep clone.png");
+        sheepDead = new Texture("Squish Sheep.png");
```

Comment:
Both of these files make the program crash because in the main repo they are renamed without any spaces. Also, the ones in the main repo are not all in one line, so the animation is broken.